packages/flutter/test/widgets/slivers_evil_test.dart

```diff
@@ -225,8 +225,8 @@ void main() {
     await tester.pump();
 
     // Screen is 600px high. Moved bottom item 500px up. It's now at the top.
-    expect(tester.getTopLeft(find.widgetWithText(DecoratedBox, '5')).dy, 0.0);
-    expect(tester.getBottomLeft(find.widgetWithText(DecoratedBox, '10')).dy, 600.0);
+    expect(tester.getTopLeft(find.widgetWithText(Container, '5')).dy, 0.0);
```

Comment:
This test was looking for an implementation detail of Container, no real need to, and that detail doesn't get created anymore in this case.

Reply:
In fairness, it was an evil test. 😄